pages/doc/logging_overview.md

```diff
@@ -153,7 +153,7 @@ Once your send your logs to Tanzu Observability, you can view all your logs on t
 See all the logs you sent to Tanzu Observability using the [Log Browser](logging_log_browser.html). You can:
 
 * See logs for a selected time range.
-  {%include note.html content="Tanzu Observability retains logs for 7, 15, or 30 days, based on your settings. To keep log data for longer, contact [technical support](https://docs.wavefront.com/wavefront_support_feedback.html#support)."%}
+  {%include note.html content="Tanzu Observability retains logs for 7, 15, or 30 days, based on your settings. For longer retention needs, contact [technical support](wavefront_support_feedback.html#support)."%}
```

Comment:
(see above)

Reply:
I will be keeping it as discussed :) There is JIRA to track this.